src/models/product-variant-model.js

```diff
@@ -150,7 +150,13 @@ const ProductVariantModel = BaseModel.extend({
       return image.variant_ids.indexOf(id) !== -1;
     })[0];
 
-    return (variantImage || primaryImage);
+    const productImage = variantImage || primaryImage;
```

Comment:
I would just call this `image`. `productImage` in this context is sorta misleading, and `image` is what we're returning.